hw06-atm/src/main/java/ru/shintar/atm/exception/InsufficientFundsException.java

```diff
@@ -0,0 +1,7 @@
+package ru.shintar.atm.exception;
+
+public class InsufficientFundsException extends Exception {
```

Comment:
Зачем проверяемые исключения?
Лучше непроверяемые.
```suggestion
public class InsufficientFundsException extends RuntimeException {
```